Add: Release announcement for 14.0 (#289)
OpenTTD/website: _posts/2024-04-01-openttd-14-0.md

It is now possible to make the calendar time walk slower, up to a point it is frozen.
This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth etc.
But it does influence vehicle introduction date, inflation, and aging.

With this feature, we hope to enable different game-plays, where people want to have more time to build out their empire before vehicles expire, inflation gets too big, etc.

A brand new day!
For some a bit slower than others.

## Ship Pathfinder

The ship pathfinder got some great love.
It is now capable of directing ships over vast distances without the need for buoys.
In fact, one could say, you don't need buoys anymore at all.
The ships will still find their way!

Finally, no more "Ship is lost" non-sense.
This enables ship-only games.

## Social Platform integration

OpenTTD 14.0 allows better integration with Social Platforms.
This is totally optional, and has to be installed separately.
But once done, OpenTTD can talk to platforms like Steam, Discord and GOG Galaxy.

For now, the functionality is limited: friends can see you play OpenTTD, if you are in a server or not, and what kind of map you are playing.

> For future versions we intend to expand on that, making more possible.

Reviewer: ```suggestion
For future versions, we intend to expand on that, making more possible.
```

But, small steps.

It has to be noted that those plugins are not licensed under GPLv2, where OpenTTD otherwise is.
Instead, they are licensed under MIT, a much more flexible license.
It is still Open Source, and the code is still freely available.
But because it integrates with closed source SDKs of these platforms, we didn't want it integrated in OpenTTD itself.
As such, they are separate plugins, and you can decide whether you want to integrate with those platforms or not!

## New players experience

First of all, there is now a Help and Manuals button in-game, that helps you through the basic information of the game.
Bringing information closer to the user, and the game more accessible to everyone, has been a path we have been on for a while.

Second, we revised a lot of settings, what their default value should be.

> Many settings have had their defaults changed, mostly towards enabling modern feature by default.

Reviewer: ```suggestion
Many settings have had their defaults changed, mostly towards enabling modern features by default.
```

---
title: OpenTTD 14.0
author: TrueBrain
---

Welcome to 14.0!

OpenTTD's first release was in March of 2004.
Now twenty years later, we are proud to present to you: release 14.0.

And boy, what a release it is.
Do I dare to say: this has been the biggest release yet?

<!-- more -->

Sometimes people try to tell us OpenTTD is dead.
Well, 14.0 shows you it really isn't.
There are so many new goodies in 14.0, that I don't even know where to start!
Let's go over a few, just to get you a feel for what to expect when you launch OpenTTD 14.0.

Read more about our birthday celebration post [here](https://www.openttd.org/news/2024/03/06/happy-birthday).

## When one day isn't a day

It is now possible to make the calendar time walk slower, up to a point it is frozen.

> This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth etc.

Reviewer 2: ```suggestion
This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth, etc.
```

## Unbunching

Do you also hate when busses start hugging each other on a route?
And you so did your best to ensure you send them out the depot with a good distance of each other.
But something must have happened down the road, and now they are very close to each other!

Introducing: unbuncing.

A new feature that allows you to tell all vehicles in a shared order pool to keep their distance; automatically, done for you.

Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/02/10/unbunching).

## Social Platform integration

OpenTTD 14.0 allows better integration with Social Platforms.
This is totally optional, and has to be installed separately.
But once done, OpenTTD can talk to platforms like Steam, Discord and GOG Galaxy.

For now, the functionality is limited: friends can see you play OpenTTD, if you are in a server or not, and what kind of map you are playing.
For future versions, we intend to expand on that, making more possible.
But, small steps.

Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/03/02/social-integration).

## Survey support

> To finally put a rest to endless debates about what setting is used and which is not, OpenTTD not has an opt-in survey tool built-in.
> Over time this will allow us, developer, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.

Reviewer 2: ```suggestion
To finally put a rest to endless debates about what setting is used and which is not, OpenTTD now has an opt-in survey tool built-in.
Over time this will allow us, developers, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.
```

You can, at any time, opt in or out from the survey.

Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/03/16/survey-https-infra)

## GUI improvements / scalable font

There has been a huge effort in making all our windows just a tiny bit better.
Fixing pixel-errors, moving things around a bit, fixing colouring issues.

But .. not less important, we now also have a built-in scalable font!
Zephyris create a real (TTF) font heavily inspired by the sprite font.
This means that scaling looks a lot better, especially for zoom levels like 1.5x.

> We love it so much, it is the default when you launch the game! (but no worries, you can revert to the sprite font if you really want to).

Reviewer 2: ```suggestion
We love it so much, it is the default when you launch the game! (But no worries, you can revert to the sprite font if you really want to.)
```
